pymess/backend/dialer/daktela.py

```diff
@@ -108,7 +114,7 @@ def publish_message(self, message):
             payload = {
                 'queue': (settings.DIALER_DAKTELA.AUTODIALER_QUEUE if message.is_autodialer
                           else settings.DIALER_DAKTELA.PREDICTIVE_QUEUE),
-                'number': message.recipient,
+                'number': re.sub(r'^\+', '00', message.recipient),
```

Comment:
recommendation from Daktela support.

Reply:
z jakého důvodu? Asi je to jedno, ale mělo by to být to samé. BTW proč to děláš regexem, to plus nazačátku by tam mělo být vždy ne?

Reply:
viz normalizace telefonu v modelu

Reply:
They said we should you this format even though it worked for me with + sign. I know when you create a user in their db you can use just digits so maybe to make it compatible...
Using regex it's IMO safer. In `normalize_phone_number` there's no check the other charaters are digits. So it could unintentionally make from non-valid phone number a valid one :) (`004201111111+11 -> 00420111111111`)

Reply:
looking at `normalize_phone_number` not every number has a plus sign - e.g. short version without country dial number.

Reply:
hmm myslel jsem, že je tam validátor na české číslo. V reálu se tam nikdy jiné nedostane. Pokud tedy bude číslo obsahovat nějaký jiný nevalidní znak tak je tato změna stejně k ničemu. Ale ok nech to být. BTW píšu česky, protože tohle stejně nikdo jiný používat nebude, tak se s tou angličtinou nemusíš tady patlat :)